homeassistant/components/islamic_prayer_times/__init__.py

```diff
@@ -41,6 +41,18 @@ def update_unique_id(
     return True
 
 
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
```

Comment:
I think I prefer the structure as described in the docs.

Reply:
done